src/main/java/homework/collections/hospital/repository/HospitalRepository.java

```diff
@@ -0,0 +1,15 @@
+package homework.collections.hospital.repository;
+
+import java.util.List;
+import java.util.Map;
+
+public interface HospitalRepository<Doctor, Patient, Time> {
```

Comment:
Rename like HospitalService.
Because we java programists write buisness logic in services